.github/ISSUE_TEMPLATE/2_bug_report_linux.yml

```diff
@@ -0,0 +1,73 @@
+name: "Report a Linux Issue"
+description: "Create a report for a runtime related Linux Issue"
+labels: ["os:linux"]
+body:
+- type: textarea
+  attributes:
+      label: "Important Information"
+      description: >
+          - mpv version
+           - Linux Distribution and Version
+           - Source of the mpv binary
+           - If known which version of mpv introduced the problem
+           - Window Manager and version
+           - GPU model, driver and version
+           - Possible screenshot or video of visual glitches
+
+          If you're not using git master or the latest release, update.
+          Releases are listed [here](https://github.com/mpv-player/mpv/releases).
+      value: |-
+          - mpv version:
+          - Linux version:
+          - Source:
+          - Introduced in version:
+          - Window Manager and version:
+          - GPU model, driver and version:
+  validations:
+      required: true
+- type: textarea
+  attributes:
+      label: "Reproduction Steps"
+      description: >
+          Try to reproduce your issue with `--no-config` first. If it isn't reproducible
+          with `--no-config` try to find out which option or script causes your issue.
+
+          If the issue is performance-related, try to reproduce it with `--no-config
+          --profile=fast`, which is designed to work on lower-end mobile devices.
+          Additionally, enable hardware decoding with `--hwdec=yes` or the `Ctrl+h` shortcut.
+
+          Describe the reproduction steps as precise as possible. It's very likely that
+          the bug you experience wasn't reproduced by the developer because the workflow
+          differs from your own.
+  validations:
+      required: true
+- type: textarea
+  attributes:
+      label: "Expected Behavior"
+  validations:
+      required: true
+- type: textarea
+  attributes:
+      label: "Actual Behavior"
+  validations:
+      required: true
+- type: textarea
+  attributes:
+      label: "Log File"
+      description: >
+          Make a log file made with `-v -v` or `--log-file=output.txt`, attach it to
+          the issue.
+
+          In the case of a crash, please provide a backtrace.
+
+          Without the log file, this issue will be closed for ignoring the issue template.
```

Comment:
i added the first 4 checkboxes for now. i am not sure how to implement the last one, so i would leave it for a later time. also removed the "do not ignore the issue template" template, since this mechanism makes it obsolete imo.

though i am not sure if we want to keep it this way.

Reply:
as an example https://github.com/Akemi/mpv/issues/26
a bit weird that it makes tasks out of it sadly.

Reply:
yeah i understand your intention. i am open for possible solutions. we probably need some extra action for that to auto-close issues if the checkbox was checked or something?

Reply:
yeah i agree with you @kasper93. if this doesn't work out well we should simplify it in the future.